java/arcs/core/data/proto/TypeProtoDecoders.kt

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2020 Google LLC.
+ *
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ *
+ * Code distributed by Google as part of this project is also subject to an additional IP rights
+ * grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+package arcs.core.data.proto
+
+import arcs.core.data.PrimitiveType
+
+fun PrimitiveTypeProto.decode(): PrimitiveType =
```

Comment:
Just a short blurb on what this extension function does would be helpful. KDoc is a requirement for all public functions, and can be especially important on extension functions given the lower amount of context available when reading the code..